uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-details-tab.component.js

```diff
@@ -379,9 +455,14 @@ define([
      */
     function initListeners () {
       listeners.push(
-        $rootScope.$on('LeaveRequestPopup::updateBalance', function () {
-          vm.updateBalance();
-          setDaySelectionMode();
+        $rootScope.$on('LeaveRequestPopup::updateBalance', function (event, absenceTypes) {
```

Comment:
and it's even worse because we then check if `absenceTypes` are there at all, meaning that sometimes they will be and sometimes they won't, making the whole thing even more confusing (why are they passed? why aren't they passed sometimes?)

Reply:
If absence period is changed we need to update entitlements in absence types. We need to pass them to avoid tick problem @deb1990 has described earlier in the ticket.